mr_proper/main.py

```diff
@@ -7,29 +8,42 @@
 
 def parse_args() -> argparse.Namespace:
     parser = argparse.ArgumentParser()
-    parser.add_argument('filename', type=str)
+    parser.add_argument('objectname', type=str)
     parser.add_argument('--recursive', action='store_true')
     return parser.parse_args()
 
 
-def main() -> None:
-    args = parse_args()
-    ast_tree = get_ast_tree(args.filename)
+def checking_function(args: argparse.Namespace) -> None:
```

Comment:
1. Too broad name. I would think about `check_file`.
2. Pass `args` is not a good idea. Lets make all used vars verbose, and keep `argparse` stuff in main.

Reply:
Done!